README.md

```diff
@@ -1,45 +1,92 @@
-| `make build-container-sensor`   | build sensor image                                                        |
-| `make build-container`          | build broker, sensor, director and example-client image                   |
-| `make build`                    | builds the director, sensor and example client binary                     |
-| `make check`                    | runs static checks                                                        |
-| `make format`                   | formats go code                                                           |
-| `make prepare`                  | installs staticcheck                                                      |
-| `make run-example-client`       | runs example client                                                       |
-| `make smoke-test`               | runs the smoke tests (builds image, start smoke tests and stop container) |
-| `make start-broker`             | starts a broker as a container                                            |
-| `make start-container`          | starts the broker, director as well as sensor container                   |
-| `make start-director`           | starts a director container                                               |
-| `make start-sensor`             | starts sensor container                                                   |
-| `make start-smoke-test`         | starts the smoke tests (broker, director, sensor and runs example client) |
-| `make stop-broker`              | stops broker container                                                    |
-| `make stop-container`           | stops running container                                                   |
-| `make stop-director`            | stops director container                                                  |
-| `make stop-sensor`              | stops a sensor container                                                  |
-| `make test`                     | runs unittests                                                            |
-| `make update`                   | updates dependencies                                                      |
-| `make`                          | default commands on make without target are: format, check, test, build   |
+# eulabeia
+
+Is management layer with the goal to make control of multiple scanner based on multiple machines easier and more predictable by introducing a small set of commands, orchestration based on a director and usage of a broker.
+
+There are four [roles](docs/roles/roles-and-relationship.md) within eulabeia:
+
+- a director - manages multiple sensors and is the main communication partner of a client
+- sensor - manages (or is) a scanner
+- scanner - a program to scan for vulnerabilities
+- client - is the using party of eulabeia
+
+The communication between each party is done via [normalized messages](docs/messaging.md) over a MQTTv5.
```

Comment:
```suggestion
The communication between each entity is done via [normalized messages](docs/messaging.md) over a MQTTv5.
```
I would use 'entity' instead of party